### PR TITLE
CryptoPro.Chromium-Gost version 108.0.5359.94

### PR DESCRIPTION
**Pull request by admtea**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90627)

### CODE REVIEW

File: manifests/c/CryptoPro/Chromium-Gost/108.0.5359.94/CryptoPro.Chromium-Gost.locale.ru-RU.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: CryptoPro.Chromium-Gost
+PackageVersion: 108.0.5359.94
+PackageLocale: ru-RU
+Publisher: CryptoPro
```
Comment (mdanish-kh) [inline manifests/c/CryptoPro/Chromium-Gost/108.0.5359.94/CryptoPro.Chromium-Gost.locale.ru-RU.yaml:7]:
```suggestion
Publisher: The Chromium-Gost Authors
```

image
This must match Control Panel/Apps&Features

File: manifests/c/CryptoPro/Chromium-Gost/108.0.5359.94/CryptoPro.Chromium-Gost.locale.ru-RU.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: CryptoPro.Chromium-Gost
+PackageVersion: 108.0.5359.94
+PackageLocale: ru-RU
+Publisher: CryptoPro
+PackageName: Chromium-Gost
+License: MIT
+Copyright: Copyright 2022 The Chromium-Gost Authors. All rights reserved.
```
Comment (mdanish-kh) [inline manifests/c/CryptoPro/Chromium-Gost/108.0.5359.94/CryptoPro.Chromium-Gost.locale.ru-RU.yaml:10]:
```suggestion
Copyright: Copyright 2022 Авторы Chromium-Gost. Все права защищены.
```
As Russian locale is used.

File: manifests/c/CryptoPro/Chromium-Gost/108.0.5359.94/CryptoPro.Chromium-Gost.locale.ru-RU.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: CryptoPro.Chromium-Gost
+PackageVersion: 108.0.5359.94
+PackageLocale: ru-RU
+Publisher: CryptoPro
+PackageName: Chromium-Gost
+License: MIT
+Copyright: Copyright 2022 The Chromium-Gost Authors. All rights reserved.
+ShortDescription: Chromium with GOST crypto algorithm support
```
Comment (mdanish-kh) [inline manifests/c/CryptoPro/Chromium-Gost/108.0.5359.94/CryptoPro.Chromium-Gost.locale.ru-RU.yaml:11]:
```suggestion
ShortDescription: Chromium с поддержкой алгоритмов ГОСТ
```
As Russian locale is used.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by mdanish-kh** [comment]
@stephengillie - PR requires manual merge.

**Pull request merged**